babel.dependencies.config.js

```diff
@@ -0,0 +1,31 @@
+/** @format */
+const config = {
+	// see https://github.com/webpack/webpack/issues/4039#issuecomment-419284940
+	sourceType: 'unambiguous',
+	presets: [
+		[
+			'@babel/env',
+			{
+				modules: false,
+				targets: { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] },
+				useBuiltIns: 'entry',
+				shippedProposals: true, // allows es7 features like Promise.prototype.finally
+			},
+		],
+		'@babel/react',
```

Comment:
This transform shouldn't be needed. We're transpiling only ES6+ syntax to ES5, nothing else.

Reply:
```suggestion
```